Add 3 `file-finder`-related features (#2825)
sindresorhus/refined-github: readme.md

- [](# "reload-failed-proxied-images") [Retries downloading images that failed downloading due to GitHub limited proxying.](https://user-images.githubusercontent.com/14858959/64068746-21991100-cc45-11e9-844e-827f5ac9b51e.png)
- [](# "forked-to") [Adds a shortcut to your forks next to the `Fork` button on the current repo.](https://user-images.githubusercontent.com/55841/64077281-17bbf000-cccf-11e9-9123-092063f65357.png)
- [](# "repo-age") [Adds the age of the repository to the statistics bar.](https://user-images.githubusercontent.com/3848317/69494069-7d2b1180-0eb7-11ea-9aa1-d4194e566340.png)

> - [](# "repo-wide-file-finder") The <kbd>t</kbd> file finder works on the `Issues` and `Pull Requests` tabs as well.

Reviewer: ```suggestion
- [](# "repo-wide-file-finder") Enables the the File Finder keyboard shortcut (<kbd>t</kbd>) on Issues and Pull Request pages as well.
```

> - [](# "file-finder-buffer") While invoking the file finder, keystrokes pressed immediately following the <kbd>t</kbd> are captured and populated in the search field.

Reviewer: ```suggestion
- [](# "file-finder-buffer") [Lets you start typing your search immediately after invoking the File Finder (<kbd>t</kbd>), instead of having you wait for it to load first.](https://user-images.githubusercontent.com/1402241/75542106-1c811700-5a5a-11ea-8aa5-bea0472c59e2.gif)
```

> - [](# "preserve-file-finder-term") [The search term is preserved across multiple invocations of the <kbd>t</kbd> file finder](https://user-images.githubusercontent.com/1402241/75542106-1c811700-5a5a-11ea-8aa5-bea0472c59e2.gif)

Reviewer: This is the wrong screenshot

```suggestion
- [](# "preserve-file-finder-term") Preserves the search terms when navigating back and forth between the File Finder and the files.
```
